pom.xml

```diff
@@ -1362,6 +1367,22 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
```

Comment:
This will only run when we run -Pdist, so did we want to include that as part of the distribution tar, or do you want this separate?